Tavisca.Bootcamp.LanguageBasics.Exercise1/Program.cs

```diff
@@ -14,16 +15,141 @@ static void Main(string[] args)
             Console.ReadKey(true);
         }
 
-        private static void Test(string args, int expected)
-        {
+        private static void Test(string args, int expected)
+        {
             var result = FindDigit(args).Equals(expected) ? "PASS" : "FAIL";
-            Console.WriteLine($"{args} : {result}");
+            Console.WriteLine($"{args} : {result}");
         }
 
-        public static int FindDigit(string equation)
-        {
-            // Add your code here.
-            throw new NotImplementedException();
+        public static int FindDigit(string equation)
+        {
+            string[] terms = equation.Split(new Char[] { '*', '=' });
+            /*The first operand has a missing digit*/
+            if (terms[0].Contains('?'))
+            {
+                int b = Int32.Parse(terms[1]);

```

Comment:
never name variable as a, b, c, x, y, z, p q, r etc.
Always name it correctly so that it is readable and self explanatory.

Reply:
Variable names updated.